rhobs/alerting/data_plane/prometheus.serviceprovider_alerts.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: rhtap-spi-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+    - name: serviceprovider_alerts
+      interval: 1m
+      rules:
+        - alert: ServiceProvider5xxErrorsRate
+          expr: |
+            (
+              sum without (status) (increase(redhat_appstudio_spi_service_provider_response_time_seconds_count{status=~"5.."}[1h]))
+              or sum without (status) (increase(redhat_appstudio_spi_service_provider_response_time_seconds_count{}[1h])) * 0
+            )
+            /
+            sum without (status) (increase(redhat_appstudio_spi_service_provider_response_time_seconds_count{}[1h])) > 0.10
+          for: 5m
+          labels:
+            severity: warning
+            slo: true
+          annotations:
```

Comment:
I suggest adding {{ $value }} to the summary/description so you have a better confidence that the query is returning what you expect it to return.

Reply:
Added